### PR TITLE
expose `rz_analysis_continue_until_syscall` `rz_analysis_continue_unt…

### PR DESCRIPTION
**Pull request by b1llow**
**Your checklist for this pull request**
- [x] I've read the [guidelines for contributing](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md) to this repository
- [x] I made sure to follow the project's [coding style](https://github.com/rizinorg/rizin/blob/master/DEVELOPERS.md#code-style)
- [x] I've documented or updated the documentation of every function and struct this PR changes. If not so I've explained why.
- [ ] I've added tests that prove my fix is effective or that my feature works (if possible)
- [ ] I've updated the [rizin book](https://github.com/rizinorg/book) with the relevant information (if needed)

**Detailed description**

**Test plan**

...

**Closing issues**

### CODE REVIEW

File: librz/include/rz_core.h [hunk 1]
```diff
@@ -759,7 +759,9 @@ RZ_API RzList *rz_core_analysis_cycles(RzCore *core, int ccl);
 RZ_API RzList *rz_core_analysis_fcn_get_calls(RzCore *core, RzAnalysisFunction *fcn); // get all calls from a function
 RZ_API void rz_core_analysis_calls(RZ_NONNULL RzCore *core, bool imports_only);
 RZ_API int rz_core_get_stacksz(RzCore *core, ut64 from, ut64 to);
-RZ_API bool rz_core_analysis_hint_set_offset(RzCore *core, const char *struct_member);
+RZ_API bool rz_core_analysis_hint_set_offset(RZ_NONNULL RzCore *core, RZ_NONNULL const char *struct_member);
+RZ_API bool rz_analysis_continue_until_syscall(RZ_NONNULL RzCore *core);
+RZ_API bool rz_analysis_continue_until_call(RZ_NONNULL RzCore *core);
```
Comment (ret2libc) [inline librz/include/rz_core.h]: use a `rz_core_` prefix

File: librz/core/canalysis.c [hunk 2]
```diff
@@ -6909,3 +6909,78 @@ RZ_API bool rz_core_analysis_hint_set_offset(RzCore *core, RZ_NONNULL const char
...
+		rz_analysis_op_free(op);
+		op = NULL;
+		if (core->analysis->esil->trap || core->analysis->esil->trap_code) {
+			break;
+		}
+	}
+	if (op) {
+		rz_analysis_op_free(op);
+		op = NULL;
+	}
```
Comment (ret2libc) [inline librz/core/canalysis.c]:
```suggestion
	rz_analysis_op_free(op);
```

File: librz/core/canalysis.c [hunk 3]
```diff
@@ -6909,3 +6909,78 @@ RZ_API bool rz_core_analysis_hint_set_offset(RzCore *core, RZ_NONNULL const char
...
```
Comment (ret2libc) [inline librz/core/canalysis.c]:
```suggestion
	rz_analysis_op_free(op);
```

**Review by ret2libc — CHANGES_REQUESTED** [review]

**Review by XVilka — APPROVED** [review]
Looks good once you address feedback by @ret2libc

**Comment by XVilka** [comment]
Btw, next time please use shorter commit/PR messages or split them into multiple lines.

**Review by XVilka — APPROVED** [review]

**Pull request merged**